packages/cli/package.json

```diff
@@ -22,7 +22,7 @@
     "prebuild": "ts-node ./scripts/generateIntlTypes.ts",
     "build:fast": "rimraf ./build && tsc --project tsconfig.build.json && yarn build:envs",
     "lint": "eslint --color -c ../../.eslintrc.js .",
-    "test": "cross-env TEST=true jest --passWithNoTests --runInBand --verbose",
+    "test": "cross-env TEST=true DOMAIN_NAME=test1.eth jest --passWithNoTests --runInBand --verbose",
```

Comment:
This probably can be improved and we can inject the env var in any other way?

Reply:
I don't think we need to use gh secrets for this. We can hardcode the envs in the workflow YAML file. There is a way to set env for jobs in gh action. secrets are only used when we want to keep envs encrypted.

Reply:
> I don't think we need to use gh secrets for this. We can hardcode the envs in the workflow YAML file. There is a way to set env for jobs in gh action. secrets are only used when we want to keep envs encrypted.

You're totally right ser @Niraj-Kamdar thanks for your thoughts!